### DISCUSSION TITLE
Mensagia Connector

### DISCUSSION THREAD

**Discussion by Sistemes-Icon** [microsoft/PowerPlatformConnectors#1478] [Q&A]
Good morning and hello to everyone! my name is Africa Santeliz and I work at Sistemes Informatics ICON, S.L.

The company Sinergyne Global Communications, S.L. hired us to create a connector for its "Mensagia" application into Power Automate.

The connector has already passed the publication phase and is already available on the Microsoft page:

Currently we want to expand the information that appears on the official page, but we do not know how we can do this process.

We are also willing to collaborate with the technical department of Sinergyne Global Communications, S.L. to remove/move the connector from the independent-publisher-connectors folder to the certified-connectors folder if necessary in order to make these modifications to the connector documentation.

Can you tell us how we could do it?

We appreciate your help.
Thanks! :)

**Comment by Amjed-Ayoub** [comment]
@AfriSR Thank you for reaching out. You can update the documentation by updating your readme.md/intro.md and submit a pull request with the updated files.
Please let me know if you have any questions.
Thank you.
